app/models/forms/bulk_upload_lettings/prepare_your_file.rb

```diff
@@ -26,11 +26,21 @@ def next_path
       end
 
       def template_path
-        "/files/bulk-upload-lettings-template-v1.xlsx"
+        "/files/bulk-upload-lettings-template-2023-24.xlsx"
+      end
+
+      def old_template_path
+        "/files/bulk-upload-lettings-template-2022-23.xlsx"
       end
 
       def specification_path
-        "/files/bulk-upload-lettings-specification-2022-23.xlsx"
+        return if year.blank?
+
+        if year <= 2022
+          "/files/bulk-upload-lettings-specification-2022-23.xlsx"
```

Comment:
```suggestion
          old_template_path
```

Reply:
Not actioning these - this is for the specification not the template, will assume you just missed that distinction :)